test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1717,38 +1717,38 @@ static Object createLargeObject(Object arg) throws Throwable {
 		return makeLargeObjectValue.invokeWithArguments(args);
 	}
 
-	static Object createAssorted(MethodHandle makeMethod, String[] fields) throws Throwable {
+	static Object createAssorted(MethodHandle makeMethod, String[] fields, boolean useDefaultValue) throws Throwable {
```

Comment:
instead of the `boolean useDefaultValue` you can just pass in the `Object[][]... initValues` with the values to be set